### PR TITLE
Add new file to prevent conflicts

### PR DESCRIPTION
**Pull request by sarankan-24**
Merge  it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# learn_git
+chandge this file # learn_git
```
Comment (kmchmk) [inline README.md]: Spelling mistakes. Please correct.

**Comment by andrewson97** [comment]
Looks good to me.

**Review by kmchmk — CHANGES_REQUESTED** [review]
Please address the comments.

File: sarankan.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+sarankan 2nd commit
```
Comment (kmchmk) [inline sarankan.txt]: Use proper English.

**Pull request closed without merging**